storagenode/piecestore/endpoint.go

```diff
@@ -74,6 +76,8 @@ type Endpoint struct {
 	orders      orders.DB
 	usage       bandwidth.DB
 	usedSerials UsedSerials
+
+	liveRequests int32
```

Comment:
we should make this be one of the first struct fields, so arm alignment is better guaranteed (https://golang.org/pkg/sync/atomic/#pkg-note-BUG)

Reply:
Wasn't this an issue for `int64` only?

Reply:
oh yeah good point, maybe worth testing on an arm device